source/jormungandr/tests/journey_common_tests.py

```diff
@@ -853,6 +853,30 @@ def test_journey_from_non_valid_object_type(self):
         assert response['error']['id'] == u'unknown_object'
         assert response['error']['message'] == u'The entry point: vehicle_journey:SNC is not valid'
 
+@dataset({"main_stif_test": {}})
+class AddErrorFieldInJormun(object):
+    def test_add_error_field(self):
+            """
+            The goal of this test is to put forward the addition of the error
+            in the Jormungandr part.
+            Kraken send journeys but after the Jormungandr filtering, there is
+            no more journeys field. So Jormungandr have to add an error field
+            with "no_solution" id and "no solution found for this journey"
+            message.
+            """
+            query = "journeys?from={from_sp}&to={to_sp}&datetime={datetime}&_override_scenario=new_default" \
```

Comment:
i'm wondering if `_override_scenario=new_default` is necessary

Reply:
True :-)